docs/version0.44.md

```diff
@@ -76,6 +77,17 @@ The `jcmd` tool also now supports specifying `0` as a VMID to target all VMs.
 
 For more information, see [Java diagnostic command (`jcmd`) tool](tool_jcmd.md).
 
+### Reference to the field names in `J9BuildFlags` changed in the DDR code
+
+The Direct Dump Reader (DDR) code enables reading system dump data by using the OpenJ9
+Diagnostic Tool Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool.
+DDR code uses fields of `J9BuildFlags` to access build flags in the system dump data.
+The names of `J9BuildFlags` fields changed over time and therefore, supporting system dumps with different build flags became a challenge.
+
+Earlier, field names in `J9BuildFlags` were based on names defined in `j9.flags`. Now, when the `J9BuildFlags` is generated for each build, the flag names are those names that are specified in `j9cfg.h` (derived from `j9cfg.h.ftl` or `j9cfg.h.in`) instead of the names that are defined in `j9.flags`. For example, `env_data64` is now referred to as `J9VM_ENV_DATA64`. Because of this change, reference to the field names in `J9BuildFlags` is changed in the DDR code as well.
```

Comment:
The final sentence here is not helpful (and is also grammatically incorrect).

Reply:
I need a sentence to link the change in `J9BuildFlags` to the corresponding change in the DDR code. Aren't `J9BuildFlags` and DDR code two different things? Trying to get the concept clear here.

Reply:
Yes, I understood that from your detailed explanation in one of your earlier comments. What I am trying to do here is create a correct flow between sentences. So, in my understanding, there are 3 steps - the names were first changed in `J9BuildFlags` (which is different from DDR code), then because of this change, the DDR code that has references to `J9BuildFlags` was changed and ultimately the expectation is that the plug-in code should also change. Is that correct?

My confusion was because of the following line:
> DDR code uses fields of `J9BuildFlags` to access build flags in the system dump data

whether when you say "...uses fields of `J9BuildFlags`" then do you mean that 

- "fields of `J9BuildFlags` is separate from DDR code and the DDR code has  flag names that are used to access those fields, or 
- that actually `J9BuildFlags` is not separate and is a part of DDR code and there are only 2 steps - one,  the DDR code that includes `J9BuildFlags` was changed and therefore the plug-in code should also change.

Reply:
From that perspective, only one thing has changed: the API that DDR plug-in code might use. The rest of the text explains why the change was made.